.isort.cfg

```diff
@@ -0,0 +1,6 @@
+[settings]
+profile=black
```

Comment:
This is the wrong profile, I'm strongly against it.

Reply:
I am going to avoid that fight and remove it. FYI, the only modification it does is on two long imports where black adds a tuple wrapper (not really bad idea).